build.gradle

```diff
@@ -40,23 +40,50 @@ repositories {
 }
 
 ext {
-    opendistroVersion = '1.7.0'
+    opendistroVersion = '1.8.0'
     isSnapshot = "true" == System.getProperty("build.snapshot", "true")
 }
 
-version = "${opendistroVersion}.0" 
+version = "${opendistroVersion}.0"
 
 apply plugin: 'elasticsearch.esplugin'
 apply plugin: 'elasticsearch.testclusters'
 apply plugin: 'jacoco'
-if (!System.properties.containsKey('tests.rest.cluster') && !System.properties.containsKey('tests.cluster')){
```

Comment:
Would this cause a failure in opendistro build? 
For example: https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/opendistro-1.7/.github/workflows/build-tar.yml#L237

Reply:
No impact. The opendistro-build doesn't run this code coverage part.
We disable the integration test coverage report and add the UT coverage report and test coverage verification as 50% as the lowest bar.